articles/aks/azure-files-csi.md

```diff
@@ -346,7 +346,10 @@ mountOptions:
 
 ### Create NFS file share storage class
 
-Create a file named `nfs-sc.yaml` and copy the manifest below. For a list of supported `mountOptions`, see [NFS mount options][nfs-file-share-mount-options]
+Create a file named `nfs-sc.yaml` and copy the manifest below. For a list of supported `mountOptions`, see [NFS mount options][nfs-file-share-mount-options].
+
+> [!NOTE]
+> `vers`, `minorversion`, `sec` have been preset by the Azure File CSI driver, and hence setting values for these options is not supported. See [discussion](https://github.com/kubernetes-sigs/azurefile-csi-driver/issues/1692). 
```

Comment:
Suggest rewording to - 

`vers`, `minorversion`, and `sec` are configured by the Azure File CSI driver. Specifying a value in your manifest for these properties aren't supported. 

I don't think it's warranted or necessary to reference the GitHub issue being we are explicitly stating that these properties are not configurable.

Reply:
@MGoedtel You are right on this. I have made proper changes on this PR.